blockstore/badger/blockstore.go

```diff
@@ -396,6 +396,9 @@ func (b *Blockstore) doCopy(from, to *badger.DB) error {
 	if workers < 2 {
 		workers = 2
 	}
+	if workers > 8 {
+		workers = 8
```

Comment:
Do we have confidence in this number? Did we check that it's "good enough" on mainnet?

Reply:
maybe 5-10 min.

Reply:
lets take a fresh measurement?

Reply:
For reference I got 50m on my fairly resourced mainnet node

Reply:
^ with current default of half CPUs
I'll measure with 8 and compare when I get to similar garbage levels

Reply:
working without major issues on mainnet (cc @TippyFlitsUK ) and this seems to be preventing out of sync on moving GC